base/arraymath.jl

```diff
@@ -244,7 +244,7 @@ end
 """
     rotr90(A, k)
 
-Rotate matrix `A` right 90 degrees an integer `k` number of times. If `k` is zero or a
+Rotate matrix `A` right 90 degrees clock-wise an integer `k` number of times. If `k` is zero or a
```

Comment:
Since we say left above, should we say right here?

Reply:
Or should we go the other way and not mention Left in the other one.
Eitehr way this  should  not say  `right 90 degrees clock-wise`